ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -596,7 +596,7 @@ public class BottomSheetController: UIViewController {
         completeAnimationsIfNeeded()
 
         if currentSheetVerticalOffset != offset(for: targetExpansionState) {
-            let animator = stateChangeAnimator(to: targetExpansionState, velocity: velocity)
+            let animator = stateChangeAnimator(to: targetExpansionState, velocity: velocity, interaction: interaction, shouldNotifyDelegate: shouldNotifyDelegate)
```

Comment:
nit: do you mind breaking these params into multiple lines? (here and line 669)

Reply:
Done